core/jazz_services/config/prod-config.json

```diff
@@ -1,8 +1,8 @@
 {
     "services_table": "{inst_stack_prefix}_services_prod",
-    "service_required_fields": ["service", "domain", "type", "created_by", "runtime", "status"],
+    "service_required_fields": ["service", "domain", "type", "created_by", "runtime", "status", "deployment_accounts"],
```

Comment:
same changes should go in dev-config.json and stg-config.json

Reply:
Added in both the json files